include/SDL3/SDL_mouse.h

```diff
@@ -83,6 +83,37 @@ typedef enum SDL_MouseWheelDirection
     SDL_MOUSEWHEEL_FLIPPED    /**< The scroll direction is flipped / natural */
 } SDL_MouseWheelDirection;
 
+/**
+ * A bitmask used when testing if a mouse's buttons are pressed.
```

Comment:
This comment needs to be updated to refer to the type, like the other flag typedefs.

Reply:
"A bitmask of pressed mouse buttons, as reported by SDL_GetMouseState, etc." ?

Reply:
Yeah, that sounds good.

Reply:
Fixed in de32eb906fde7549822fc5a928f1280807945f97.